article-analyzer/src/main/java/co/inventorsoft/academy/spring/service/ArticleService.java

```diff
@@ -0,0 +1,33 @@
+package co.inventorsoft.academy.spring.service;
+
+import co.inventorsoft.academy.spring.model.Article;
+import co.inventorsoft.academy.spring.repository.ArticleRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+public class ArticleService {
+    private final ArticleRepository articleRepository;
+    private final List<String> excludedWords;
+
+    public List<List<String>> getSplitArticleWords() {
+        return articleRepository.getArticles()
+                .stream()
+                .map(Article::getContent)
+                .map(content -> Arrays.stream(content.split("\\s+"))
+                        .map(word -> word.replaceAll("\\p{Punct}+", "").toLowerCase())
```

Comment:
![JackSparrowWhereDidYouGetThatGIF](https://github.com/vitaliiStefanchakInventorsoft/spring-lecture/assets/103140969/c5a9beea-39b5-4cf3-862c-79b0c9455022)

Reply:
Don't quite understad what do you mean

Reply:
have seen similar code, but that's fine